### PR TITLE
issue #64: fix PeerSet operations with IpBlock + update calico testca…

### PR DESCRIPTION
**Pull request by adisos**
…se24

### CODE REVIEW

**Review by zivnevo — CHANGES_REQUESTED** [review]
just these 2 small things

File: tests/calico_testcases/example_policies/testcase24-selector-expression/testcase24-c.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: projectcalico.org/v3
+kind: NetworkPolicy
+metadata:
+  name: testcase24-a
```
[the remark this answers is not in the document]
Comment (adisos) [reply tests/calico_testcases/example_policies/testcase24-selector-expression/testcase24-c.yaml]: done

File: tests/calico_testcases/example_policies/testcase24-selector-expression/testcase24-new-scheme.yaml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+
+queries:
+  - name: policies_not_vacuous
+    vacuity:
+    - np_a
+    - np_b
+    expected: 0
+
+  - name: nonEquiv
```
[the remark this answers is not in the document]
Comment (adisos) [reply tests/calico_testcases/example_policies/testcase24-selector-expression/testcase24-new-scheme.yaml]: done

**Review by zivnevo — APPROVED** [review]
ok to merge

**Pull request merged**